api/middleware/sendData.js

```diff
@@ -0,0 +1,15 @@
+const axios = require('axios');
+const DSURL = process.env.DS_API_URL;
```

Comment:
We are already accessing the `DS_API_URL` environment variable in `config/dsConfig.js`. Could we just import it from there?

Reply:
Fixed!